TESTING.md

```diff
@@ -0,0 +1,141 @@
+# Keylime testing
+
+## Pull-request testing
+
+Individual changes to keylime are introduced using the pull-request workflow and for each pull-request update a set of test is executed.
+
+### Keylime testsuite
```

Comment:
Do you want to add also a reference to the `.ci/run_local.sh` script?

Reply:
@THS-on Would you suggest some wording? I am not familiar with it so I do not want to add something misleading.

Reply:
The Keylime test suite can be run locally by executing `.ci/run_local.sh` (requires Docker).

Reply:
I have added Local execution section at the very end of the doc, please review.